content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx

```diff
@@ -86,181 +88,115 @@ schema.sql
 serverless.yml
 ```
 
-## 2. Set the DATABASE_URL environment variable locally
-
-Set the `DATABASE_URL` environment variable locally so you can create the database schema and Prisma can access the database to introspect:
-
-```terminal
-export DATABASE_URL="postgresql://__USER__:__PASSWORD__@__HOST__/__DATABASE__"
-```
+## 2. Set the DATABASE_URL in <inlinecode>.env</inlinecode>
 
-> **Note:** you will need the `DATABASE_URL` environment variable for the subsequent steps. Set it in all terminal sessions related to this project.
+The Lambda functions need the `DATABASE_URL` environment variable to access the database. You must define an `.env` file. The preconfigured `serverless-dotenv-plugin` uses this file to inject into the function runtime.
 
-You need to replace the uppercase placeholders with your database credentials, e.g.:
+The repository contains an `.env.example` example file to help with this.
 
-```terminal
-postgresql://janedoe:randompassword@yourpostgres.compute-1.amazonaws.com:5432/yourdbname
-```
+### Steps
 
-## 3. Set the DATABASE_URL in `.env`
+1. Enter the following command to copy `.env.example` to a new file, `.env`:
 
-For the Lambda functions to access the database, they need access to the `DATABASE_URL` environment variable.
+   ```terminal
+   cp .env.example .env
+   ```
 
-For this, you need to define a `.env` file which the preconfigured `serverless-dotenv-plugin` will use to inject into the function runtime.
+2. Open `.env` and set the `DATABASE_URL` to your database credentials. The format is as follows:
 
-The repository contains a `.env.example` example file to assist with this.
+```file
```

Comment:
Indent all lines after 2. so that they are nested under step 2.

Reply:
Agreed